cloud/scope/machinepool.go

```diff
@@ -124,6 +124,10 @@ func (m *MachinePoolScope) ScaleSetSpec() azure.ScaleSetSpec {
 
 // Name returns the Azure Machine Pool Name.
 func (m *MachinePoolScope) Name() string {
+	// Windows Machine pools names cannot be longer than 9 chars
```

Comment:
is this a VMSS restriction or a Windows hostname restriction? asking because I noticed for Machines it's 15 chars

Reply:
VMSS only, This is becuase VMSS adds unique value to the end of each instance